diff -Nur mod_python-3.0.1.orig/Makefile.in mod_python-3.0.1/Makefile.in
--- mod_python-3.0.1.orig/Makefile.in	Fri Nov  8 01:15:10 2002
+++ mod_python-3.0.1/Makefile.in	Thu Feb 27 09:48:44 2003
@@ -92,7 +92,8 @@
 	@echo
 	@echo "Performing DSO installation."
 	@echo
-	$(INSTALL) src/mod_python.so $(LIBEXECDIR)
+	$(INSTALL) -d $(DESTDIR)$(LIBEXECDIR)
+	$(INSTALL) src/mod_python.so $(DESTDIR)$(LIBEXECDIR)
 	@$(MAKE) install_py_lib
 	@echo
 	@echo "Now don't forget to edit your main config and add"
@@ -102,13 +103,13 @@
 	@echo
 
 install_py_lib:
-	$(INSTALL) -d $(PY_STD_LIB)/site-packages/mod_python
+	$(INSTALL) -d $(DESTDIR)$(PY_STD_LIB)/site-packages/mod_python
 	@for f in `ls lib/python/mod_python/*.py`; \
 	do \
-		$(INSTALL) $$f $(PY_STD_LIB)/site-packages/mod_python; \
+		$(INSTALL) $$f $(DESTDIR)$(PY_STD_LIB)/site-packages/mod_python; \
 	done
 	${PYTHON_BIN} $(PY_STD_LIB)/compileall.py $(PY_STD_LIB)/site-packages/mod_python
-	${PYTHON_BIN} -O $(PY_STD_LIB)/compileall.py $(PY_STD_LIB)/site-packages/mod_python
+	${PYTHON_BIN} -O $(PY_STD_LIB)/compileall.py $(DESTDIR)$(PY_STD_LIB)/site-packages/mod_python
 
 clean:
 	cd src && $(MAKE) clean
