--- mod_python-3.0.3/Makefile.in.orig	2003-06-11 22:17:54.386314384 +0000
+++ mod_python-3.0.3/Makefile.in	2003-06-11 22:18:17.801754696 +0000
@@ -93,7 +93,7 @@
 	@echo
 	@echo "Performing DSO installation."
 	@echo
-	$(INSTALL) src/mod_python.so $(LIBEXECDIR)
+	$(INSTALL) src/mod_python.so $(DESTDIR)$(LIBEXECDIR)
 	@$(MAKE) install_py_lib
 	@echo
 	@echo "Now don't forget to edit your main config and add"
