diff -Nur mod_python-3.1.0a.orig/dist/setup.py.in mod_python-3.1.0a/dist/setup.py.in
--- mod_python-3.1.0a.orig/dist/setup.py.in	Thu May 29 16:15:46 2003
+++ mod_python-3.1.0a/dist/setup.py.in	Wed Oct  8 01:17:53 2003
@@ -40,7 +40,7 @@
                                   "src/psp_parser.c",
                                   "src/_pspmodule.c"],
                                  include_dirs=["src/include"],
-                                 libraries=["l"]
+                                 libraries=["fl"]
                                  )
                        ]
           )
diff -Nur mod_python-3.1.0a.orig/src/Makefile.in mod_python-3.1.0a/src/Makefile.in
--- mod_python-3.1.0a.orig/src/Makefile.in	Thu Aug 14 17:24:42 2003
+++ mod_python-3.1.0a/src/Makefile.in	Wed Oct  8 01:17:44 2003
@@ -63,7 +63,7 @@
 # requires flex 2.5.31 for reentrant support
 LEX=/usr/local/bin/flex
 INCLUDES=@INCLUDES@
-LIBS=@LIBS@ -ll
+LIBS=@LIBS@ -lfl
 LDFLAGS=@LDFLAGS@
 OPT=
 CFLAGS=$(OPT) $(INCLUDES) 
