--- configure.in	Fri Nov  1 01:13:47 2002
+++ configure.in1	Fri Nov  1 01:12:46 2002
@@ -254,7 +254,8 @@
 
 PyLIBP=${PyEXEC_INSTALLDIR}/lib/python${PyVERSION}
 PyLIBPL=${PyLIBP}/config
-PyPYTHONLIBS=${PyLIBPL}/libpython${PyVERSION}.a
+#PyPYTHONLIBS=${PyLIBPL}/libpython${PyVERSION}.a
+PyPYTHONLIBS=/usr/lib/libpython${PyVERSION}.a
 PyLIBS=`grep "^LIB[[SMC]]=" ${PyLIBPL}/Makefile | cut -f2 -d= | tr '\011\012\015' '   '`
 PyMODLIBS=`grep "^LOCALMODLIBS=" ${PyLIBPL}/Makefile | cut -f2 -d= | tr '\011\012\015' '   '`
 PY_LIBS="${PyPYTHONLIBS} ${PyLIBS} ${PyMODLIBS}"
