--- mod_python-2.6.4/configure.in.wiget	Tue Dec 12 19:08:13 2000
+++ mod_python-2.6.4/configure.in	Tue Dec 12 19:13:21 2000
@@ -187,7 +187,7 @@
 if test -z "$PYTHON_SRC"; then
   PyLIBP=${PyEXEC_INSTALLDIR}/lib/python${PyVERSION}
   PyLIBPL=${PyLIBP}/config
-  PyPYTHONLIBS=${PyLIBPL}/libpython${PyVERSION}.a
+  PyPYTHONLIBS="-L${PyLIBPL} -lpython${PyVERSION}"
   PyLIBS=`grep "^LIB[[SMC]]=" ${PyLIBPL}/Makefile | cut -f2 -d= | tr '\011\012\015' '   '`
   PyMODLIBS=`grep "^LOCALMODLIBS=" ${PyLIBPL}/Makefile | cut -f2 -d= | tr '\011\012\015' '   '`
   PY_LIBS="${PyPYTHONLIBS} ${PyLIBS} ${PyMODLIBS}"
