--- mod_python-3.0.3/Makefile.in.orig	2003-06-11 22:36:03.877686520 +0000
+++ mod_python-3.0.3/Makefile.in	2003-06-11 22:36:41.091029232 +0000
@@ -104,13 +104,11 @@
 
 install_py_lib:
 	cd dist && $(MAKE) install_py_lib
-#	$(INSTALL) -d $(PY_STD_LIB)/site-packages/mod_python
-#	@for f in `ls lib/python/mod_python/*.py`; \
-#	do \
-#		$(INSTALL) $$f $(PY_STD_LIB)/site-packages/mod_python; \
-#	done
-#	${PYTHON_BIN} $(PY_STD_LIB)/compileall.py $(PY_STD_LIB)/site-packages/mod_python
-#	${PYTHON_BIN} -O $(PY_STD_LIB)/compileall.py $(PY_STD_LIB)/site-packages/mod_python
+	$(INSTALL) -d $(DESTDIR)$(PY_STD_LIB)/site-packages/mod_python
+	@for f in `ls lib/python/mod_python/*.py`; \
+	do \
+		$(INSTALL) $$f $(DESTDIR)$(PY_STD_LIB)/site-packages/mod_python; \
+	done
 
 clean:
 	cd src && $(MAKE) clean
