--- mod_python-3.0.3/src/include/mod_python.h~	2003-09-14 20:11:08.000000000 +0000
+++ mod_python-3.0.3/src/include/mod_python.h	2003-09-14 20:12:54.000000000 +0000
@@ -138,6 +138,11 @@
 #define SILENT 0
 #define NOTSILENT 1
 
+/* python 2.3 no longer defines LONG_LONG, it defines PY_LONG_LONG */
+#ifndef LONG_LONG
+#define LONG_LONG PY_LONG_LONG
+#endif
+
 /* structure to hold interpreter data */
 typedef struct {
     PyInterpreterState *istate;
