diff -Naur mod_python-2.7.8.org/src/Makefile.in mod_python-2.7.8/src/Makefile.in
--- mod_python-2.7.8.org/src/Makefile.in	Sun Jun 22 13:30:21 2003
+++ mod_python-2.7.8/src/Makefile.in	Sun Jun 22 13:30:43 2003
@@ -52,7 +52,7 @@
 INCLUDES=@INCLUDES@
 LIBS=@LIBS@
 LDFLAGS=@LDFLAGS@
-OPT=
+OPT=@OPT@
 CFLAGS=$(OPT) $(INCLUDES) 
 srcdir=.
 
@@ -119,4 +119,4 @@
 serverobject.o: serverobject.c
 connobject.o: connobject.c
 
-# DO NOT DELETE THIS LINE
\ No newline at end of file
+# DO NOT DELETE THIS LINE
diff -Naur mod_python-2.7.8.org/configure.in mod_python-2.7.8/configure.in
--- mod_python-2.7.8.org/configure.in	Sun Jun 22 13:41:45 2003
+++ mod_python-2.7.8/configure.in	Sun Jun 22 13:42:01 2003
@@ -301,6 +301,8 @@
   AC_PATH_PROG(MKDEP, "makedepend")
 fi
 
+AC_SUBST(OPT, $OPT)
+
 AC_OUTPUT(Makefile src/Makefile src/libpython.module Doc/Makefile)
 
 if test -n "$MKDEP"; then
