diff -urNp -x '*.orig' mod_python-3.5.0.org/src/_apachemodule.c mod_python-3.5.0/src/_apachemodule.c
--- mod_python-3.5.0.org/src/_apachemodule.c	2013-11-12 04:21:34.000000000 +0100
+++ mod_python-3.5.0/src/_apachemodule.c	2021-03-02 22:05:35.930030347 +0100
@@ -836,7 +836,7 @@ static struct PyModuleDef _apache_module
 
 PyObject *_apache_module_init()
 {
-    PyObject *m, *d, *o;
+    PyObject *m, *d, *o, *mod;
 
     PyType_Ready(&MpTable_Type);
     PyType_Ready(&MpTableIter_Type);
@@ -851,7 +851,7 @@ PyObject *_apache_module_init()
 #else
     m = PyModule_Create(&_apache_moduledef);
     PyObject *name = PyUnicode_FromString("_apache");
-    _PyImport_FixupExtensionObject(m, name, name);
+    _PyImport_FixupExtensionObject(m, name, name, mod);
 #endif
     d = PyModule_GetDict(m);
     Mp_ServerReturn = PyErr_NewException("_apache.SERVER_RETURN", NULL, NULL);
